src/todos/todos.service.ts

```diff
@@ -22,4 +22,11 @@ export class TodosService {
     }
     return foundTodo;
   }
+
+  async updateTodoById(id: string, updateTodo: UpdateTodoDto): Promise<Todo> {
+    const foundTodo = await this.getTodoById(id);
+    this.todoRepository.assign(foundTodo, updateTodo);
+    await this.todoRepository.getEntityManager().flush();
```

Comment:
In my opinion, this is a case where a repository method (like updateOne) makes sense as the builtin method names are not exactly clear